frontend/src/composables/http/useUnauthorizedMiddleware.ts

```
@@ -0,0 +1,22 @@
+import { useAuthToken } from '@/composables';
+import type { HttpMiddleware } from './HttpMiddlewareExecutor';
+import { HttpErrorReason } from './HttpErrorReason';
+
+export function useUnauthorizedMiddleware(): HttpMiddleware {
+  const router = useRouter();
+  const authToken = useAuthToken();
+
+  return {
+    async interceptResponseError(error): Promise<void> {
+      if (error.reason === HttpErrorReason.UNAUTHORIZED) {
+        delete authToken.value;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)